### ISSUE TITLE
Build Error: Cannot get textContent on a raw TW_Element

### ISSUE THREAD

**Issue by tobibeer** [Jermolene/TiddlyWiki5#3025]
After using `npm link` to build a demo using the current master and then reinstalling the latest release via `npm install -g tiddlywiki` fast forwarding my build env from 5.1.13 to 5.1.15, I am now getting this error when trying to build my plugin documentation at https://github.com/tobibeer/tw5-plugins:

```
$:/core/modules/utils/fakedom.js:222
                        throw "Cannot get textContent on a raw TW_Element";
                        ^
Cannot get textContent on a raw TW_Element
```

Any clue where that might have its cause?

The last time I successfully built [tw5-plugins](http://tobibeer.github.io/tw5-plugins) was with [5.1.13](http://tobibeer.github.io/tw5-plugins/#%24%3A%2FControlPanel).

Although quite probably of no use, here's the powershell I used so far:

```
git checkout gh-pages
git merge origin/master
cd editions/plugins
Write-Host "Build TW"
tiddlywiki --build
cd ../library
Write-Host "Build Plugin Library"
tiddlywiki --output "../../library/" --build library
git add -A :/
git commit -m "update pages"
git push
git checkout master
cd ../../
Write-Host "Press any key to continue ..."
$x = $host.UI.RawUI.ReadKey("NoEcho,IncludeKeyDown")
```

**Comment by pmario** [comment]
which nodejs version do you use?

**Comment by tobibeer** [comment]
Will have to check. (Not at my machine atm). (Always) Safe to update to latest version of nodejs?

**Comment by pmario** [comment]
I would go with LTS. ... But it could also be a problem with a missing `;` in js, or a `,` too much in json config. ... That's the common problems I create testing stuff :)

**Comment by tobibeer** [comment]
Gwiw, I can't spot any syntax issues with [tiddlywiki.info|https://github.com/tobibeer/tw5-plugins/blob/master/editions/plugins/tiddlywiki.info].

**Comment by Jermolene** [comment]
Hi @tobibeer that error suggests that the system is trying to do something with the fakedom that doesn't work (but presumably does work when you run it in the browser).

I notice that you're building all the build targets. Is it possible that you're trying to build a static representation, and you're using a plugin that doesn't support static rendering?

**Comment by tobibeer** [comment]
Thanks @Jermolene. Turns out, at some point, it seems I wanted to add **getElementById()** support to fakedom.js and actually commited it to my [tw5-dom](tobibeer.github.io/tw5-dom) repo. This one then conflicted with expectations for the current version of fakedom.js. So, for now I updated the custom version to correspond to the new one, but I guess I will have to do away with the custom one, or rather make a PR of it should I deem it fit for survival.
